bin/dump-taprc-schema.js

```diff
@@ -0,0 +1,95 @@
+const { options, reporters } = require("./jack.js")
```

Comment:
This file probably better belongs in `scripts`, since it's a standalone script, not part of the `bin` unit.

Reply:
> I think the deploy preview you want is [deploy-preview-601--node-tap.netlify.com](https://deploy-preview-601--node-tap.netlify.com/)


I meant those links in the Checks section of this PR
https://app.netlify.com/sites/node-tap/deploys/5db3961926859e0008ac8a80

They only for a brief moment show something and then the whole site goes black.